job_spec/SRG_GSLC.yml

```diff
@@ -30,14 +30,22 @@ SRG_GSLC:
       command:
         - ++process
         - back_projection
+        - --gpu
         - --bucket
         - '!Ref Bucket'
         - --bucket-prefix
         - Ref::bucket_prefix
         - Ref::granules
       timeout: 10800
       vcpu: 1
-      memory: 31500
+      gpu: 1
+      memory: 30500
       secrets:
         - EARTHDATA_USERNAME
         - EARTHDATA_PASSWORD
+  compute_environment:
+    name: SrgGslc
+    instance_types:
+      - g6.2xlarge
+    # Image ID for: /aws/service/ecs/optimized-ami/amazon-linux-2/recommended/image_id
+    ami_id: ami-0ca2af66da8e56876
```

Comment:
I'd suggest moving the `compute_environment` block to just before the `tasks` block (just after `cost_profiles`) to make it visually more obvious that `compute_environment` applies to the entire job spec and is not part of the individual task specification.